docs/quick-start/environment.mdx

```diff
@@ -56,7 +56,7 @@ Visit your app on `http://localhost:3000`. You can interact with your smart cont
 - Edit your deployment scripts:
   - Hardhat => `packages/hardhat/deploy`
   - Foundry => `packages/foundry/script`
-- Edit your frontend in `packages/nextjs/pages`
+- Edit your frontend in `packages/nextjs/app/page.tsx` (homepage), and the rest of [pages / layouts](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts) for the different [routes](https://nextjs.org/docs/app/building-your-application/routing/defining-routes) of your application.
```

Comment:
"and the rest of page / layout ..." seems a bit confusing to me 😅 

Maybe : 

"Edit your frontend homepage at `packages/nextjs/app/page.tsx`,  checkout next docs on how [routing](https://nextjs.org/docs/app/building-your-application/routing/defining-routes) and configuring [pages / layout](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts) works"

Lol not sure if I made it more complicated. but please feel free to update / improve it🙌

Reply:
Love your suggestions, just pushed the change! 🙌